WordPressAuthenticator/GoogleSignIn/OAuthRequestBody+GoogleSignIn.swift

```diff
@@ -16,7 +16,7 @@ extension OAuthTokenRequestBody {
             clientSecret: "",
             audience: audience,
             code: authCode,
-            codeVerifier: pkce.codeVerifier,
+            codeVerifier: pkce.codeVerifier.rawValue,
```

Comment:
nit: Could this be opaque, and only require passing the `pkce.codeVerifier`, and not having to see the `.rawValue`?

Reply:
Yes, that would be a neat refinement.

Reply:
> Yes, that would be a neat refinement.

...which I might tackle in a dedicated PR because it would require changing the shape of a few types. The `codeVerifier` input in the request object currently maps to the `code_verifier` JSON string. Changing that will require some plumbing.

Reply:
https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/745